Source/BugsnagConfiguration.m

```diff
@@ -475,8 +475,8 @@ - (void)setMaxBreadcrumbs:(NSUInteger)capacity {
  */
 - (BOOL)shouldRecordBreadcrumbType:(BSGBreadcrumbType)type {
     // enabledBreadcrumbTypes is BSGEnabledBreadcrumbTypeNone
-    if (!self.enabledBreadcrumbTypes) {
-        return YES;
+    if (self.enabledBreadcrumbTypes == 0 && type != BSGBreadcrumbTypeManual) {
```

Comment:
A comparison to `BSGEnabledBreadcrumbTypeNone` would be more explicit here.

Reply:
Updated comparison to use `BSGEnabledBreadcrumbTypeNone`